fuels-abigen-macro/tests/test_projects/large-return-data/src/main.sw

```diff
@@ -3,25 +3,65 @@ contract;
 use std::*;
 use core::*;
 use std::storage::*;
+use std::contract_id::ContractId;
+
+pub struct SmallStruct {
+    foo: u32,
+}
+
+pub struct LargeStruct {
+    foo: u8,
+    bar: u8,
+}
 
 abi TestContract {
-  fn get_id() -> b256;
-  fn get_small_string() -> str[8];
-  fn get_large_string() -> str[9];
+    fn get_id() -> b256;
+    fn get_small_string() -> str[8];
+    fn get_large_string() -> str[9];
+    fn get_large_struct() -> LargeStruct;
+    fn get_small_struct() -> SmallStruct;
+    fn get_large_array() -> [u32;
+    2];
```

Comment:
https://github.com/FuelLabs/sway/issues/726 strikes again. cc @eureka-cpu

Reply:
Likely what's happening is the formatter is adding a newline character when it finds a `;` and this edge case wasn't taken into account. Currently working on the `format stucts and enums` draft that I had yet to finish, so will take a look at this too.